pkg/controller/llmisvc/DEV.md

```diff
@@ -437,6 +556,8 @@ kind: Gateway
 metadata:
   name: openshift-ai-inference
   namespace: openshift-ingress
+  labels:
+    kuadrant.io/gateway: "true"
```

Comment:
Should this be moved to another section (if we need one) that covers kuadrant/rhcl setup?

Reply:
ok I will remove this from here and will create another section for authZ.